token/cli/src/main.rs

```diff
@@ -7425,4 +7453,39 @@ mod tests {
                 .unwrap()
         );
     }
+
+    #[tokio::test]
+    #[serial]
+    async fn metadata_pointer() {
+        let (test_validator, payer) = new_validator_for_test().await;
+        let program_id = &spl_token_2022::id();
+        let config = test_config_with_default_signer(&test_validator, &payer, &program_id);
```

Comment:
Clippy's complaining about this, you can just do
```suggestion
        let program_id = spl_token_2022::id();
        let config = test_config_with_default_signer(&test_validator, &payer, &program_id);
```